console/src/main/java/org/eclipse/rdf4j/console/util/ConsoleQueryResultWriter.java

```diff
@@ -0,0 +1,137 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.console.util;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.rdf4j.common.text.StringUtil;
+
+import org.eclipse.rdf4j.console.ConsoleIO;
+import org.eclipse.rdf4j.console.Util;
+
+import org.eclipse.rdf4j.model.Value;
+
+import org.eclipse.rdf4j.query.BindingSet;
+import org.eclipse.rdf4j.query.QueryResultHandlerException;
+import org.eclipse.rdf4j.query.TupleQueryResultHandlerException;
+import org.eclipse.rdf4j.query.resultio.AbstractQueryResultWriter;
+import org.eclipse.rdf4j.query.resultio.QueryResultFormat;
+
+/**
+ * Write query results to console
+ * 
+ * @author Bart Hanssens
+ */
+public class ConsoleQueryResultWriter extends AbstractQueryResultWriter {
```

Comment:
I wonder if we should move this class to query.resultio (or one of its helper packages), as it may be useful outside of the console tool.

Reply:
could be, in that case I'd try to rework it a bit so that it depends on an OutputStream only, instead of directly using the ConsoleIO.write

Reply:
I'd suggest to merge it first as it is in dev (I'd like to add other stuff on namespaces, but that will be easier when this PR is merged), and fine-tune it afterwards.

E.g. another nice-to-have thingie would be a command to read an existing srj/srx or ttl/... file and display it in the Console